lib/persisted-model.js

```diff
@@ -1421,11 +1417,13 @@ PersistedModel.rectifyAllChanges = function(callback) {
  * @param {Error} err Error object; see [Error object](http://docs.strongloop.com/display/LB/Error+object).
  */
 
-PersistedModel.handleChangeError = function(err) {
-  if (err) {
-    console.error(Model.modelName + ' Change Tracking Error:');
-    console.error(err);
-  }
+PersistedModel.handleChangeError = function(err, operation) {
+  if (!err) return;
+
+  console.error('%s Change Tracking Error%s:',
```

Comment:
Or with `this.emit('error', err).` We should pass the "operation" value to the event too.